src/pythium/sklimming/reader.py

```diff
@@ -51,9 +71,14 @@ def make_sample_path(locations: List[Path], tags:List[str]) -> List[str]:
     Return:
         paths: A list of full paths up to sample tags
     '''
-    paths = [str(loc)+'/'+tag for tag in tags for loc in locations]
-    dirpaths = [p+'/.*'  for path in paths for p in glob.glob(path) if os.path.isdir(p)]
-    fpaths = list(set([path  for path in paths for p in glob.glob(path) if not os.path.isdir(p) ]))
+    if remote_glob_imported:
```

Comment:
if you implement the comment on impotts, I believe you should have only one set of these lines. Less boilerplate is always better, less chances for errors and easier to edit in the future.